packages/analytics-js/src/constants/logMessages.ts

```diff
@@ -257,6 +257,9 @@ const INVALID_POLYFILL_URL_WARNING = (
 const BAD_COOKIES_WARNING = (key: string) =>
   `The cookie data for ${key} seems to be encrypted using SDK versions < v3. The data is dropped. This can potentially stem from using SDK versions < v3 on other sites or web pages that can share cookies with this webpage. We recommend using the same SDK (v3) version everywhere or avoid disabling the storage data migration.`;
 
+const PAGE_UNLOAD_ON_BEACON_DISABLED_WARNING = (): string =>
+  `Page Unloaded event is only supported for Beacon transport mechanism. Please set useBeacon to true to enable this feature.`;
```

Comment:
We should include context similar to other messages.

Reply:
Converted to a static string

Reply:
It is not enough. The context is needed.
Something similar to this,

```
${context}${LOG_CONTEXT_SEPARATOR}Page Unloaded event can only be tracked when the Beacon transport is active. Please enable "useBeacon" load API option.
```

Reply:
Addressed